session_2/student_registry/db/migrate/20210123124824_create_students.rb

```diff
@@ -0,0 +1,14 @@
+class CreateStudents < ActiveRecord::Migration[6.1]
+  def change
+    create_table :students do |t|
+      t.string :name
+      t.string :roll_number
+      t.string :branch
+      t.decimal :cgpa, precision: 4, scale: 2
```

Comment:
Precision - Number of digits.
Scale - Number of digits after the decimal point.

Since CGPA has two digits after the decimal point and lies between 0 and 10, it will have scale of 2 and precison of 4.

But precision of 4 implies that `12.34` is valid CGPA, after it has 4 digits (precision) and 2 digits after decimal (scale).

Therefore, I have added a validation in `app/models/student.rb` to limit CGPA between 0 and 10.